static/js/info_overlay.js

```diff
@@ -149,6 +150,135 @@ This text won't be visible until the user clicks.
     },
 ];
 
+const search_operator_rows = [
+    {
+        operator_key: "stream",
+        operator_value: "stream",
+        effect: "Narrow to messages on stream",
+    },
+    {
+        operator_key: "topic",
+        operator_value: "topic",
+        effect: "Narrow to messages with topic",
+    },
+    {
+        operator_key: "pm-with",
+        operator_value: "email",
+        effect: "Narrow to private messages with",
+    },
+    {
+        operator_key: "group-pm-with",
+        operator_value: "email",
+        effect: "Narrow to group private messages with",
```

Comment:
Read https://zulip.readthedocs.io/en/latest/translating/internationalization.html for background, but because different languages have different word order, you can't write it like this.  Instead you want something like this approach we use in JS templates:

"Narrow to group private messages with <z-link>email</z-link>" 

which translators can translate as 

"Words in <z-link>emailtranslation</z-link> different order"

and then our code code render the example as `group-pm-with:emailtranslation`.

Reply:
Oh! Thanks, I have reflected the newer changes in the latest push. Hope that resolve these issues :)